lcls_tools/superconducting/scLinac.py

```diff
@@ -480,39 +480,48 @@ def move(
         maxSteps: int = utils.DEFAULT_STEPPER_MAX_STEPS,
         speed: int = utils.DEFAULT_STEPPER_SPEED,
         changeLimits: bool = True,
+        check_detune: bool = True,
     ):
         """
         :param numSteps: positive for increasing cavity length, negative for decreasing
         :param maxSteps: the maximum number of steps allowed at once
         :param speed: the speed of the motor in steps/second
         :param changeLimits: whether to change the speed and steps
+        :param check_detune: whether to check for valid detune after each move
         :return:
         """
 
         self.check_abort()
+        maxSteps = abs(maxSteps)
 
         if changeLimits:
             # on the off chance that someone tries to write a negative maximum
```

Comment:
Is a negative no longer a problem?

Reply:
This just immediately turns max steps into a positive number regardless of what the user requested

Reply:
So this is the max steps _possible_ for all cavities, which is some default in utils.DEFAULT_STEPPER_MAX_STEPS? Is the default sometimes neg?

Reply:
Lisa clarified, it's the maxSteps for one tuner movement.